docs/java-application-development/development-environment-setup.md

```diff
@@ -1,183 +1,161 @@
 # Development Environment Setup
 
-This document describes how to set up the development environment for Kura, which consists of the following components:
-
-* JVM (Java JDK SE 8)
-
-* Eclipse IDE
-
-* Kura Workspace setup
+In this document we'll cover the required steps to setup the Development Environment for contributing to the Eclipse Kura project. If, instead you want to develop applications or bundles running on Kura refer to the [Kura Workspace setup guide](./kura-workspace-setup.md).
```

Comment:
> running on Kura refer to the [Kura Workspace setup guide]

running on Eclipse Kura refer to the Eclipse [Kura Workspace setup guide]

Reply:
Addressed in 07b2d2eb82c3658a32602d3b85954f5ac5c6d23e